integration-tests/gatsby-source-wordpress/test-fns/test-utils/queries.js

```diff
@@ -536,10 +536,6 @@ exports.queries = {
       opengraphType
       title
       twitterDescription
-      twitterImage {
-        id
-        title
-      }
```

Comment:
`twitterImage` seems to throw errors on latest WPGraphQL Yoast when you don't have an image set. Since we're panicking on errors I'm omitting this field